### PR TITLE
dependency lovin ❤️

### PR DESCRIPTION
**Pull request by palmenhq**
Here's some upgrades of dependencies, including emotion 10. Sorry for the massive PR, but i really didn't change much except the compatibility issues 🙈

### CODE REVIEW

File: src/blocks/ImageTextBlock/index.tsx [hunk 1]
```diff
@@ -28,75 +28,62 @@ import { BackgroundVideo } from './BackgroundVideo'
...
-        : 'row',
-    justifyContent: 'space-between',
-    alignItems: 'center',
-    [TABLET_BP_DOWN]: {
-      flexDirection: 'column',
-    },
-  }),
-)
+const AlignableContentWrapper = styled(ContentWrapper)<{
+  textPosition: string
```
Comment (hedvigoscar) [inline src/blocks/ImageTextBlock/index.tsx]: Should this use the `TextPosition`-type? 🙂

File: src/blocks/ImageTextBlock/index.tsx [hunk 2]
```diff
@@ -107,68 +94,56 @@ const Paragraph = styled('div')(
...
-  }: {
-    alignment: string
-    displayOrder: 'top' | 'bottom'
-    hasLink?: boolean
-  }) => ({
-    width: hasLink ? '100%' : '40%',
-    flexShrink: hasLink ? 1 : 0,
+const ImageVideoWrapper = styled('div')<{
+  alignment: string
+  displayorder: 'top' | 'bottom'
```
Comment (hedvigoscar) [inline src/blocks/ImageTextBlock/index.tsx]: This also changed case 🤔

File: src/components/GlobalStyles.tsx [hunk 3]
```diff
@@ -1,82 +1,83 @@
+import { css } from '@emotion/core'
 import { colors, fonts, getCdnFontFaces } from '@hedviginsurance/brand'
-import * as React from 'react'
-import { injectGlobal } from 'react-emotion'
 
-export const GlobalStyles: React.FunctionComponent = () => {
-  // tslint:disable-next-line no-unused-expression
-  injectGlobal`
-${getCdnFontFaces()}
+export const globalStyles = () => css`
```
Comment (hedvigoscar) [inline src/components/GlobalStyles.tsx]: Does this need to be a function? 🙂

File: src/blocks/AnimatedPhoneBlock.tsx [hunk 4]
```diff
@@ -50,14 +50,14 @@ const TextWrapper = styled('div')(
...
-  ({ displayOrder }: { displayOrder: 'top' | 'bottom' }) => ({
+const Title = styled('h2')<{ displayorder: 'top' | 'bottom' }>(
+  ({ displayorder }) => ({
     fontSize: '4.5rem',
     wordWrap: 'break-word',
     width: '100%',
     [TABLET_BP_DOWN]: {
       fontSize: '3.75rem',
-      marginTop: displayOrder === 'top' ? '3rem' : '1.414rem',
+      marginTop: displayorder === 'top' ? '3rem' : '1.414rem',
```
[the remark this answers is not in the document]
Comment (palmenhq) [reply src/blocks/AnimatedPhoneBlock.tsx]: yes, new react complains about the property `displayOrder` for some reason :)

File: src/blocks/ImageTextBlock/index.tsx (hunk 1 above)
Comment (palmenhq) [reply src/blocks/ImageTextBlock/index.tsx]: same about this one, react complains about the camel case and suggests this name 🤷‍♀️

File: src/components/GlobalStyles.tsx (hunk 3 above)
Comment (palmenhq) [reply src/components/GlobalStyles.tsx]: no, but i like the immutability

File: .babelrc.js [hunk 5]
```diff
@@ -8,6 +8,8 @@ module.exports = {
     'react-hot-loader/babel',
     '@babel/plugin-syntax-dynamic-import',
     '@babel/plugin-proposal-class-properties',
+    '@babel/plugin-proposal-optional-chaining',
+    '@babel/plugin-proposal-nullish-coalescing-operator',
```
[the remark this answers is not in the document]
Comment (palmenhq) [reply .babelrc.js]: Yes, it does since 1.19 which i've upgraded it to :)

File: src/components/GlobalStyles.tsx (hunk 3 above)
Comment (hedvigoscar) [reply src/components/GlobalStyles.tsx]: Will it not be evaluated once at module evaluation time and then be constant?

File: src/blocks/AnimatedPhoneBlock.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (hedvigoscar) [reply src/blocks/AnimatedPhoneBlock.tsx]: Fair enough, we can disregard all of these 🙂

File: src/components/GlobalStyles.tsx (hunk 3 above)
Comment (palmenhq) [reply src/components/GlobalStyles.tsx]: yes, but you can still modify it after. I guess its a matter of preference, i like the thought of not being able to modify stuff. But i can make it mutable

Comment (hedvigoscar) [reply src/components/GlobalStyles.tsx]: It's not super-important to me. I think as a rule if someone starts mutating the output of a module we have to bust them in PRs anyways as that's highly illegal 👮

Comment (palmenhq) [reply src/components/GlobalStyles.tsx]: haha yes that's fair

**Pull request merged**